### PR TITLE
62 take tokens design

### LINKED ISSUE #62: Take tokens design

### PR DESCRIPTION
**Pull request by KamilDevelopments**
I added panel used for taking tokens + fixed cardViews in GameActivity

### CODE REVIEW

**Comment by kestivvi** [comment]
It should be possible to set negative amount of tokens you want to get. Because exchange should be possible when you have over 10 tokens.

**Comment by KamilDevelopments** [comment]
Okay I've added option to remove up to 3 tokens, as this is the maximum amount you can take in 1 turn

File: app/src/main/java/com/example/splendormobilegame/MainActivity.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ private void setupButtons() {
             @Override
             public void onClick(View v) {
                 //create new intent -> pass class -> start intent
-                Intent myIntent = new Intent(MainActivity.this, CreateRoomActivity.class);
+                Intent myIntent = new Intent(MainActivity.this, GameActivity.class);
```
Comment (kestivvi) [inline app/src/main/java/com/example/splendormobilegame/MainActivity.java]: Ok, change it back to CreateRoomActivity and we can merge it.

**Review by kestivvi — CHANGES_REQUESTED** [review]

**Review by kestivvi — APPROVED** [review]
Brilliant ✌️✌️

**Review by Vertonowsky — APPROVED** [review]

**Pull request merged**